src/main/resources/global/90-init-alfresco.sh

```diff
@@ -23,6 +34,33 @@ if [[ $SOLR_SSL == none ]] && [[ $ALFRESCO_VERSION != "5.0"* ]] && [[ $ALFRESCO_
   xmlstarlet edit --inplace --delete "/Server/Service[@name='Catalina']/Connector[@SSLEnabled='true' and @port='\${TOMCAT_PORT_SSL}']" $TOMCAT_SERVER_FILE
 fi
 
+# initialiaze variables for keystore location
+DIR_KEYSTORE=${DIR_KEYSTORE:-'/opt/alfresco/keystore'}
```

Comment:
Must be quoted in case the path contains spaces

```suggestion
DIR_KEYSTORE="${DIR_KEYSTORE:-'/opt/alfresco/keystore'}"
```